dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/WorkloadDriver.java

```diff
@@ -141,7 +141,7 @@ public static void main(String[] args) throws Exception {
 
   private Class<? extends WorkloadMapper> getMapperClass(String className) throws ClassNotFoundException {
     if (!className.contains(".")) {
-      className = WorkloadDriver.class.getPackage().getName() + "." + className;
+      className = WorkloadDriver.class.getPackage().getName() + ".audit." + className;
```

Comment:
This will fix things for the `AuditReplayMapper`, but break things for `CreateFileMapper`... I created the `audit` package to group together related classes, but didn't realize I broke the assumption that all mapper classes would be in the same package. I was trying to be clever here, but maybe it would be better to just maintain a static mapping of short name -> class.